modules/ting_search_context/Gemfile

```diff
@@ -0,0 +1,9 @@
+# Bundler configuration file
+
+source 'https://rubygems.org'
+
+group :frontend do
+  gem 'compass',        '~> 0.12.5'
+  gem 'sass',           '~> 3.2.19'
+  gem 'zen-grids',      '~> 1.4'
+end
```

Comment:
Gemfile, Gemfile.lock &config.rb works as documentation of tools, but it would have been preferable if libSass + compass-mixins had been used